proto/toit/model/job.proto

```diff
@@ -91,6 +94,14 @@ message CronSchedule {
   google.protobuf.Timestamp valid_to = 8;
 }
 
+message JobPubSub {
+  message Subscription {
+    toit.model.pubsub.Topic topic = 1;
+    bytes id = 2;
```

Comment:
Let's see if we can make this one local - perhaps even job status?

Reply:
Talking offline - we decided not to for now. Will be deprecated once we get something more goalstate driven.